app/models/casa_case.rb

```diff
@@ -86,7 +86,7 @@ def update_cleaning_contact_types(args)
   def clear_court_dates
     update(court_date: nil,
            court_report_due_date: nil,
-           court_report_submitted: false) if court_date.present? && court_date < Time.now
+           court_report_submitted: false) if court_date < Time.current
```

Comment:
Oh wait, sorry if I was unclear -- we still need to check that the court date exists:

```
if court_date && court_date < Time.current
```

Reply:
oh lol. whoops. I maybe misread. Apologies.